src/tests/ftest/util/general_utils.py

```diff
@@ -1,5 +1,6 @@
 """
   (C) Copyright 2018-2024 Intel Corporation.
+  (C) Copyright 2025 Hewlett Packard Enterprise Development LP
```

Comment:
Shouldn't this PR also replace the https://github.com/daos-stack/daos/blob/ffd8f2deec38115b7b2137b8ea55335fcb076633/src/tests/ftest/util/general_utils.py#L721 pcmd call in this file?

Reply:
Oops. I overlooked the usage in both dump_engines_stacks and get_host_data